plugin-server/src/config/config.ts

```diff
@@ -131,9 +131,9 @@ export function getDefaultConfig(): PluginsServerConfig {
         SESSION_RECORDING_LOCAL_DIRECTORY: '.tmp/sessions',
         // NOTE: 10 minutes
         SESSION_RECORDING_MAX_BUFFER_AGE_SECONDS: 60 * 10,
-        SESSION_RECORDING_MAX_BUFFER_SIZE_KB: ['dev', 'test'].includes(process.env.NODE_ENV || 'undefined')
-            ? 1024 // NOTE: ~1MB in dev or test, so that even with gzipped content we still flush pretty frequently
-            : 1024 * 50, // ~50MB after compression in prod
+        SESSION_RECORDING_MAX_LINES_PER_FILE: ['dev', 'test'].includes(process.env.NODE_ENV || 'undefined')
```

Comment:
Personally I don't like this. We _only_ have this so we avoid writing super big files. By basing this on line count we are really not sure at all anymore if it is based on file size which begs the question - why have it at all 🤔 

Feels like we are more likely to end up flushing a file that just has a lot of tiny mutations rather than flushing what we are actually targeting...

Reply:
tbh I almost removed it instead of changing it... looking in prod we flush c500 lines per file, but we see (very infrequent) peaks of tens of thousands.

And even then we almost never flush by size. 

So, let's remove it. It's noise that isn't helping